test/micro/org/openjdk/bench/java/util/ArraysMismatch.java

```diff
@@ -62,8 +62,8 @@
     public void setup() {
         leftStartRange = size / 4;
         leftEndRange = size - size / 4;
-        rightStartRange = size / 4 + 10;
-        rightEndRange = size - size / 4 + 10;
+        rightStartRange = size / 4 + 1;
```

Comment:
Since you changed `10` to `1` perhaps make this a parameter defaulting to the new value?

Reply:
I have reverted the changes in existing benchmark since a new benchmark has been added for partial in-lining cases. PR has been updated with its performance results.